client/src/pages/Login/login.css

```diff
@@ -0,0 +1,14 @@
+.center {
```

Comment:
You can also mvoe this to index.css, para wla nay .css files sa mga folder lain

Reply:
should I move the home.css file to index.css as well?